src/arch/xtensa/configs/imx8_defconfig

```diff
@@ -2,3 +2,5 @@ CONFIG_IMX8=y
 CONFIG_HAVE_AGENT=n
 CONFIG_FORMAT_CONVERT_HIFI3=n
 CONFIG_KPB_FORCE_COPY_TYPE_NORMAL=n
+CONFIG_MATH_DECIBELS=y
+CONFIG_NUMBERS_NORM=y
```

Comment:
should we make CONFIG_DRC select this two configs?

Reply:
Yeah, DRC should probably select these directly. @singalsu do we have any other processing components that would have a similar relationship with maths library logic ?

Reply:
This was a good catch. I missed this because Intel DMIC selected these, sorry! Yep, the DRC should select these.